crates/sdk/macros/src/logic/method.rs

```diff
@@ -113,7 +113,12 @@ impl<'a> ToTokens for PublicLogicMethod<'a> {
             Some(type_) => {
                 let def = match type_ {
```

Comment:
seems to me, this match statement is now irrelevant?

Reply:
Should we just `.unwrap()` then? If we can guarantee that the apps state was initialized.